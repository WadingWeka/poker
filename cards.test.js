const card = require("./cards")

describe("check card generation", () => {
    test('2 of clubs', () => {
        expect(card.assembleCard(2,0)).toMatchSnapshot();
    });

    test('10 of hearts', () => {
        expect(card.assembleCard(10,2)).toMatchSnapshot();
    });

    test('King of spades', () => {
        expect(card.assembleCard(13,3)).toMatchSnapshot();
    });
    test('Ace of diamonds', () => {
        expect(card.assembleCard(1,1)).toMatchSnapshot();
    });
});

