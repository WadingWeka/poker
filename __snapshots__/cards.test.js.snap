// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check card generation 2 of clubs 1`] = `
Array [
  "┌──────────────┐",
  "│ 2♣           │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│            2♣│",
  "└──────────────┘",
]
`;

exports[`check card generation 10 of hearts 1`] = `
Array [
  "┌──────────────┐",
  "│10♥           │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│           10♥│",
  "└──────────────┘",
]
`;

exports[`check card generation Ace of diamonds 1`] = `
Array [
  "┌──────────────┐",
  "│ A♦           │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│            A♦│",
  "└──────────────┘",
]
`;

exports[`check card generation King of spades 1`] = `
Array [
  "┌──────────────┐",
  "│ K♠           │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│              │",
  "│            K♠│",
  "└──────────────┘",
]
`;
